iidm/iidm-modification/src/test/java/com/powsybl/iidm/modification/ReplaceThreeWindingsTransformersBy3TwoWindingsTransformersTest.java

```diff
@@ -55,6 +56,21 @@ void replaceBasicTest() {
         assertEquals(3, network.getTwoWindingsTransformerCount());
     }
 
+    @Test
+    void replaceSelectedThreeWindingsTransformerTest() {
```

Comment:
Same here: could you also add a test with another ID to assert that the replacement is not performed in this case?

Reply:
Done